parts/linux/cloud-init/artifacts/components.json

```diff
@@ -360,6 +360,29 @@
       "multiArchVersions": [
         "v0.7.1"
       ]
+    },
+    {
+      "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
+      "amd64OnlyVersions": [],
+      "multiArchVersions": [
```

Comment:
what's the rule for those version to be all cached ?  we will need to find a way to ensure we only cached what is really needed.